src/code-templates/libraries/logger/test/index.test.ts

```diff
@@ -7,77 +7,71 @@ beforeEach(() => {
 });
 
 describe('logger', () => {
-  test('When no explicit configuration is set, info logs are written', async () => {
-    // Arrange
-    const stdoutStub = sinon.stub(process.stdout, 'write');
+  let stdoutStub: sinon.SinonStubbedMember<typeof process['stdout']['write']>;
 
+  beforeEach(() => {
```

Comment:
Do we really need two hirearchical beforeEach? One is simpler

Reply:
agree